scripts/bootstrap/compile.sh

```diff
@@ -21,7 +21,10 @@ if [ -d derived/jars ]; then
 else
   PROTOBUF_JARS="/usr/share/java/protobuf.jar /usr/share/java/protobuf-java-util.jar"
 fi
-PROTO_FILES=$(find third_party/remoteapis third_party/googleapis third_party/pprof src/main/protobuf src/main/java/com/google/devtools/build/lib/buildeventstream/proto src/main/java/com/google/devtools/build/skyframe src/main/java/com/google/devtools/build/lib/skyframe/proto src/main/java/com/google/devtools/build/lib/bazel/debug src/main/java/com/google/devtools/build/lib/starlarkdebug/proto -name "*.proto")
+
+# Parse third_party/googleapis/BUILD.bazel to find the proto files we need to compile from googleapis
+GOOGLE_API_PROTOS="$(grep -o "\".*\.proto\"" third_party/googleapis/BUILD.bazel | sed "s/\"//g" | sed "s/^/third_party\/googleapis\//g")"
```

Comment:
Can we make this a little bit more readable, e.g. like this?
```
grep -o '".*\.proto"' third_party/googleapis/BUILD.bazel | sed 's/"//g' | sed 's|^|third_party/googleapis/|g'
```

Reply:
Nice, thanks!